src/tests/units/FakeMyElefant.php

```diff
@@ -14,7 +14,7 @@ public function getAuthentification($secretKey){
         return false;
     }
 
-    public function sendSms(string $secretToken, array $contacts, $sendDate=null, $message = null, $sender = null){
+    public function sendSms($secretToken, $contacts, $sendDate=null, $message = null, $sender = null){
         if ($secretToken === 'validToken') {
```

Comment:
Is this a mistake? `'validToken'`?

Reply:
Here is the FakeMyElefant.php so that's fine, I think :)